respeaker_ros/scripts/speech_to_text.py

```diff
@@ -2,24 +2,46 @@
 # -*- coding: utf-8 -*-
 # Author: Yuki Furuta <furushchev@jsk.imi.i.u-tokyo.ac.jp>
 
+from __future__ import division
+
 import actionlib
 import rospy
 try:
     import speech_recognition as SR
 except ImportError as e:
     raise ImportError(str(e) + '\nplease try "pip install speechrecognition"')
 
+import numpy as np
 from actionlib_msgs.msg import GoalStatus, GoalStatusArray
 from audio_common_msgs.msg import AudioData
+from audio_common_msgs.msg import AudioInfo
 from sound_play.msg import SoundRequest, SoundRequestAction, SoundRequestGoal
 from speech_recognition_msgs.msg import SpeechRecognitionCandidates
 
 
 class SpeechToText(object):
     def __init__(self):
         # format of input audio data
-        self.sample_rate = rospy.get_param("~sample_rate", 16000)
-        self.sample_width = rospy.get_param("~sample_width", 2)
+        if rospy.get_param('~audio_info', None):
```

Comment:
```
if rospy.get_param('~audio_info', false):
```
is more readable?

Reply:
https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/350/commits/296cc6ae3c326ec4cdf6c8f8487155017fb56ffb
I modified the code to make it more readable.
I look forward to your feedback.

Reply:
In this case, `'~audio_info'` rosparam only takes string variable.
I think it's good.

Reply:
Thank you!